athenet/sparsifying/sharpen_filters.py

```diff
@@ -0,0 +1,90 @@
+import numpy
+import cv2
+
+from athenet.layers.conv import ConvolutionalLayer
+
+
+def get_noise_indicators(filter, bilateral_filter_args):
+    """
+    Returns possibility of being a noise .
+
+    This function, for 2D filter with values in (-1, 1),
+    computes possibility of being a noise,
+    which is computed using bilateral filtering with given arguments
+
+    :param numpy.ndarray filter: 2D filter
+    :param bilateral_filter_args: args for bilateral filtering
+    :type bilateral_filter_args: list or tuple
+    """
+
+    filter_as_image = numpy.array(filter * 255, dtype=numpy.uint8)
+    sharpened_filter = cv2.bilateralFilter(filter_as_image,
+                                           *bilateral_filter_args)
+
+    return (numpy.array(sharpened_filter, dtype=numpy.float32) -
+            numpy.array(filter_as_image, dtype=numpy.float32)) / 255.
+
+
+def sharpen_filter(filter, min_noise_indicator, max_value,
+                   bilateral_filter_args):
```

Comment:
So what would you suggest? 5 elements in this tuple? I have like that previously and it was very inconvenient for me